litmus-portal/frontend/src/views/ChaosWorkflows/BrowseAnalytics/CustomStatus/styles.ts

```diff
@@ -0,0 +1,50 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  passed: {
+    width: '4.75rem',
+    textAlign: 'center',
+    borderRadius: 3,
+    paddingTop: theme.spacing(0.375),
+    paddingBottom: theme.spacing(0.375),
+    color: theme.palette.primary.dark,
+    backgroundColor: theme.palette.customColors.menuOption.active,
+    verticalAlign: 'middle',
+    display: 'inline-flex',
+  },
+  failed: {
```

Comment:
CSS for passed and failed are same, only backgroundColor is different, It can avoided by defining a separate class for common parameters and just concatenating to it the different one.
You can use the  `${} ${}`  sysntax. cc: @arkajyotiMukherjee